public/controllers/management/components/management/status/status-overview.js

```diff
@@ -131,15 +135,16 @@ export class WzStatusOverview extends Component {
             3000
           );
         } else {
+          const managerInfo = await this.statusHandler.managerInfo();
           const daemons = await this.statusHandler.managerStatus();
-          const listDaemons = this.objToArr(daemons.data.data.affected_items[0]);
+          const listDaemons = this.objToArr(daemons?.data?.data?.affected_items?.[0]);
+          const managerInfoData = managerInfo?.data?.data?.affected_items?.[0];
           this.props.updateListDaemons(listDaemons);
           this.props.updateSelectedNode(false);
-          this.props.updateNodeInfo((managerInfo.affected_items || [])[0] || {});
```

Comment:
**question**: I see you added an API request to get the manager's information. I don't see this request was done in the previous code. Was it missing this request?

Reply:
Before it was done on line [100](https://github.com/wazuh/wazuh-kibana-app/pull/5528/files/a3db04120b3c88c33ef77ed2bbe62444e3902e9b#diff-1706a17dd7de80ea73834fa9449298e0ef7dd2b77dd1308bf8d4333cd6d6d44eL100), and it was only used if it was in manager mode, so what I did was to make the request when it needs it.